pkgs/top-level/all-packages.nix

```
@@ -2881,6 +2881,8 @@ in
   tsm-client = callPackage ../tools/backup/tsm-client { jdk8 = null; };
   tsm-client-withGui = callPackage ../tools/backup/tsm-client { };
 
+  trac = pythonPackages.callPackage ../tools/misc/trac { };
```

would be nice nice for upstream to have a python3 supported installation....
```suggestion
  trac = pythonPackages.callPackage ../tools/misc/trac { };
```

---

Yeah, sadly not supported.